docs/tutorials/feature-matrix.rst

```diff
@@ -3,27 +3,28 @@ Feature Matrix
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Nesting                   | yes      | no             | yes      | read-only | write-only | flattened | yes       |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Transitions               | yes      | yes            | planned  | read-only | write-only | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Audio/Video Effects       | planned  | needs research | planned  | planned   | planned    | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Speed Effects             | yes      | yes            | planned  | read-only | planned    | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Color Decision List (CDL) | metadata | yes            | planned  | planned   | planned    | planned   | no        |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
\ No newline at end of file
++--------------------------+----------+-------+--------+--------+-------+--------+---------+----+
+|Feature                   | OTIO   | EDL     |FCP7 XML| FCP X  | AAF   |   RV   | ALE     |Gst |
++==========================+==========+=======+========+========+=======+========+=========+====+
+|Single Track of Clips     |    ✔   |    ✔    |    ✔   |   ✔    |   ✔   | W-O    |    ✔    | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Multiple Video Tracks     |    ✔   |    ✖    |    ✔   |   ✔    |   ✔   | W-O    |flattened| ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Audio Tracks & Clips      |    ✔   | testing |    ✔   |   ✔    |   ✔   | W-O    |    ✔    | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Gap/Filler                |    ✔   |    ✔    |    ✔   |   ✔    |   ✔   |   ✔    |skipped  | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Markers                   |    ✔   |    ✔    |    ✔   |contrib?|planned|contrib?|   ✖     | ✔  |
++--------------------------+--------+---------+--------+-----------+-------------+---------+----+
+|Nesting                   |    ✔   |    ✖    |    ✔   |   ✔    |   ✔   |   W-O  |flattened| ✔  |
++--------------------------+------------------+--------+--------+-------+--------+---------+----+
+|Transitions               |    ✔   |    ✔    |contrib?|contrib?|   ✔   |   W-O  |   ✖     | ✔  |
```

Comment:
Looks like FCPX doesn't have transition support (I Don't see any mention of them in the code).  Looks like FCP7 XML does have transition support based on the code.